src/analyze/analysis_handler.go

```diff
@@ -34,7 +35,7 @@ func AnalyzeBGPMessages(window common.Window, config *config.Configuration) comm
 
 	//put all the results into the Result struct and pass write it out to a json
 	r := common.Result{
-		WindowSize: config.WindowSize,
+		WindowSize: strconv.Itoa(config.WindowSize),
```

Comment:
I thought we dedcided to make this an int?

Reply:
^ this version of analysis handler is outdated - taya will be updating it to be an int in a future PR
for now casting to a string since value was changed to an int in config